ci/official/utilities/setup.sh

```diff
@@ -92,7 +93,7 @@ fi
 # we want to make sure those extra jobs don't upload anything.
 # TODO(angerson) Remove this once it's no longer relevant
 if [[ "${KOKORO_BUILD_INITIATOR:-}" != "kokoro" ]]; then
-  source ./ci/official/envs/disable_all_uploads
+  source ./ci/official/envs/no_upload
 fi
```

Comment:
WDYT about adding in, either by default or by option, the ability to print out the resolved list of TFCI variables.  I think it could be helpful in debugging and AFAIK we don't plug any secrets into anything TFCI_* but i could see that being a concern.